ui/src/VoxettaApp.js

```diff
@@ -30,3 +26,36 @@ export class VoxettaApp extends LitElement {
 }
 
 customElements.define('vox-app', VoxettaApp);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
```

Comment:
This functionality should go into it's own file, wrapped up in a class or closure such as `AudioRecorder'. Then the `vox-app` component can create an instance of the class and call the methods it wants when the time is right. This audio recording functionality should be de-coupled from the DOM, not depending on event listeners or specific buttons being present on the DOM.

Class could looks something like this

```javascript
export class AudioRecorder {

  startRecording() {
    // logic to start recording
  }
  stopRecording() {
    // logic to stop recording
  }
  getRecordedAudio() [
    // logic to get out the recorded Blob audio
    // maybe this class would also serve as a memory storage container for audio?
  }
}
```

With class audio recorder, the LitElement can just call `audioRecorder.startRecording`, `audioRecorder.stopRecording` in its event handlers.